src/main/java/org/spongepowered/api/Game.java

```diff
@@ -75,6 +77,15 @@
     GameRegistry getRegistry();
 
     /**
+     * Gets an instance of the provided {@link Provider} class.
+     *
+     * @param providerClass The class of the type of {@link Provider}
+     * @param <T> The type of {@link Provider}
+     * @return An instance of the requested provider type
+     */
+    <T extends Provider> T getProvider(Class<T> providerClass);
```

Comment:
Because technically a provider is contained in the registry, but as it was
already convoluted to get the game registry and then retrieve the instance.

By using providers it makes more sense to get them from the Game instance.
However it still made some sense to leave them with the registry itself. I
think I'll remove it. I meant to ask about opinions for the inclusion there
but I forgot.